src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  font-family: "Roboto", sans-serif;
```

Comment:
Check `checklist.md` paragraph 2.

You must use fallback fonts **(alternative font)**